addons/web/static/src/js/widgets/weekly_recurrent_task.js

```diff
@@ -0,0 +1,87 @@
+odoo.define('web.weekly_recurrent_task', function (require) {
```

Comment:
The file name, module name, Component name and tag should be the same (written differently though, e.g. snake case, camel case). I suggest "WeekDays", because the widget can be used in other scenarios than a recurring one. Basically, it simply maps week days to boolean values. So
  module: "web.WeekDays"
  Component: "WeekDays"
  file and key in registry: "week_days"